lib/snapshot/runner.rb

```diff
@@ -18,6 +18,13 @@ def work
 
       clear_previous_screenshots if Snapshot.config[:clear_previous_screenshots]
 
+      Helper.log.info "Resetting all simulators screen scale to 1".green
+
+      all_device_types = `xcrun simctl list devicetypes`.scan(/.*\s\((.*)\)/)
```

Comment:
@KrauseFx Done. Should I update https://github.com/fastlane/snapshot/blob/master/lib/snapshot/reset_simulators.rb#L29 as well?